docs/workshop-hello-world.md

```diff
@@ -0,0 +1,127 @@
+# Purpose
+This is meant to be a small workshop that explores deploying a hello world application
+to the AWS EKS Kubernetes cluster, as well as exploring a few items available within
+the cluster.
+
+## Pre-Req
+- Install a tool to access the cluster in a user friendly way such as: https://k9scli.io/
+- Set up a SSO login session to access the cluster and login, for example: https://sagebionetworks.jira.com/wiki/spaces/IT/pages/2632286259/AWS+SSM+Session+Manager#%5CuD83D%5CuDCD8-Log-in-to-an-AWS-account
+  - `aws sso login --profile dnt-dev-admin`
+- Update your kubeconfig to connect to the cluster:
```

Comment:
Can you elaborate on what kubeconfig is and what purpose it serves?

Reply:
Added link to https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/ and snippet:
`- Update your [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) to connect to the cluster. The kubeconfig in this case is referring to the authentication mechanism used to access the cluster.`